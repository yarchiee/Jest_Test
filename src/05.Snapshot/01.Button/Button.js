export default function Button() {
  return <button>Haha</button>;
}
