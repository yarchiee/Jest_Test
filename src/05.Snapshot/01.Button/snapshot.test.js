import { create } from "react-test-renderer";
import Button from "./Button";

it("snapshot of button", () => {
  const tree = create(<Button />).toJSON();
  expect(tree).toMatchSnapshot();
});
