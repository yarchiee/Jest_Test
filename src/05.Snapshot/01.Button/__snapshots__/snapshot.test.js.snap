// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot of button 1`] = `
<button>
  Haha
</button>
`;
